Date: Feb 5, 2016 9:37 AM

From: 560bf904ed7c707bd45e474d825e80d71638fcb4 Feb 5, 2016 12:54 PM

Subject: [PATCH] RDKSYSINTSW-1306: Fix out of tree build for libupnp

        This is windows specific code. It assumes that
        build is happening in same dir as sources. This
        breaks out of tree builds.

        We dont have windows case, so just delete it

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: Khem Raj <khem_raj@cable.comcast.com>
---

Index: libupnp-1.6.19/ixml/Makefile.am
===================================================================
--- libupnp-1.6.19.orig/ixml/Makefile.am
+++ libupnp-1.6.19/ixml/Makefile.am
@@ -7,7 +7,7 @@
 
 SUBDIRS 		= doc
 
-AM_CPPFLAGS		= -I$(srcdir)/inc -I$(srcdir)/src/inc
+AM_CPPFLAGS		= -I$(top_srcdir)/ixml/inc -I$(top_srcdir)/ixml/src/inc -I$(top_srcdir)/upnp/inc
 AM_CFLAGS		= 
 
 LDADD			= libixml.la
Index: libupnp-1.6.19/threadutil/Makefile.am
===================================================================
--- libupnp-1.6.19.orig/threadutil/Makefile.am
+++ libupnp-1.6.19/threadutil/Makefile.am
@@ -4,7 +4,7 @@
 # (C) Copyright 2005 Remi Turboult <r3mi@users.sourceforge.net>
 #
 
-AM_CPPFLAGS		= -I$(srcdir)/inc -I$(srcdir)/src/inc
+AM_CPPFLAGS		= -I$(top_srcdir)/threadutil/inc -I$(top_srcdir)/threadutil/src/inc -I$(top_srcdir)/upnp/inc
 
 if ENABLE_DEBUG
     AM_CPPFLAGS		+= -DDEBUG -DSTATS
